crosnest/chains.json

```diff
@@ -253,6 +253,15 @@
         "run_time": "23:00",
         "minimum_reward": 1000000
       }
+    },
+    {
+      "name": "nomic",
+      "address": "nomic1xewp986hcaudlx04epgpcpu556tte28lsd7t6z",
+      "restake": {
```

Comment:
You should leave out the `restake` section if you don't offer it - not 100% what this will do otherwise